contracts/PercentageTransferManager/.ink/abi_gen/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "abi-gen"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
Do we need to clone the `abi_gen` package here?

Reply:
This has come from the cargo-contract template. It should work as it is.